ntoskrnl/mm/ARM3/virtual.c

```diff
@@ -5107,22 +5107,13 @@ NtFreeVirtualMemory(IN HANDLE ProcessHandle,
     BOOLEAN Attached = FALSE;
     PAGED_CODE();
 
-    //
```

Comment:
Keep the previous style with empty comment lines before and after

Reply:
Reverted.